examples/garrawarla/configs/compilers.yaml

```diff
@@ -0,0 +1,28 @@
+compilers:
+- compiler:
+    spec: gcc@4.8.5
+    paths:
+      cc: /usr/bin/gcc
+      cxx: /usr/bin/g++
+      f77: /usr/bin/gfortran
+      fc: /usr/bin/gfortran
+    operating_system: sles12
+    target: x86_64
+    modules: []
+    environment: {}
+    extra_rpaths: []
+    flags: {}
+- compiler:
+    spec: gcc@10.1.0
```

Comment:
Just noting that the default GCC compiler on Garrawarla is currently 8.3.0.
For instance, OpenMPI on Garrawarla is currently not built with 10.1.0.
If users and us are happy with using 10.1.0 for the Spack stack, then no further action here.